src/lib/Libifl/pbsD_asyrun.c

```diff
@@ -49,24 +49,22 @@
 
 
 /**
- * @brief
- *	-send async run job batch request.
+ * @brief	Helper function for pbs_asynrunjob and pbs_asynrunjob_ack
```

Comment:
Can't we also update __pbs_runjob() to use the helper function. There is no difference between __pbs_runjob and pbs_asyrunjob_ack other than the batch request ID.